RELEASE_NOTES.md

```diff
@@ -79,12 +154,12 @@ For a list of all deprecated CSM features, see [Deprecations](introduction/depre
 
 * [BOS](glossary.md#boot-orchestration-service-bos) v1
 
-For a list of all features with an announced removal target, see [Removals](introduction/deprecated_features/README.md#removals).
+For a list of all features with an announced removal target,
+see [Removals](introduction/deprecated_features/README.md#removals)
 
 ## Known issues
```

Comment:
I think we should call out the postgres failover issue here.

troubleshooting/known_issues/postgres_database_recovery.md

Neither workaround fully solves the problem, an upgrade of the postgresql components is needed and whenever a master node is rebooted or rebuilt there is a chance that postgresql will failover without the clients noticing resulting in services behaving strangely.

Reply:
I added this information to the document.